docs/contribution/releases/building-and-publishing.md

```diff
@@ -0,0 +1,48 @@
+   1. Run from the workflow from trunk and enter the major version number.
+   2. Once the workflow finishes, it should’ve created 2 new PRs:
+      * One against trunk removing processed entries.
+      * One against the release branch, both removing processed entries and updating the readme.txt with the updated changelog.
+   3. Review and merge the PRs, updating if necessary.
+      * Ensure that the date in the changelog is also correct.
+2. (Skip for `-rc.1`) Run the version-bump workflow to update the WooCommerce version in relevant files on the release branch ahead of the release:
+   * This command will give you a link to a PR. Review and merge the PR once CI passes.
+3. Build the release ZIP file by running the [“Release: Build ZIP file” workflow](https://github.com/woocommerce/woocommerce/actions/workflows/release-build-zip-file.yml).
+   1. Run the workflow from the release branch and set "Create a draft GitHub release" to `true`.
+   2. Once the workflow is complete, the run will create a [draft release tag](https://github.com/woocommerce/woocommerce/releases) with an attached woocommerce.zip file.
+
+## Publishing the Release
+
+1. Run the ["Release: Upload release to WordPress.org" workflow](https://github.com/woocommerce/woocommerce/actions/workflows/release-upload-to-wporg.yml) to publish the release on WordPress.org
+   * Run the workflow from `trunk` and enter the tag of the release that was created above, e.g. 10.0.1.
+   * This step will create a new tag in SVN with the new release version and, if this release is a newer release than the current version in trunk, it will overwrite trunk. For example, if the latest trunk version is 10.1.0 and release 10.0.4 is published, it will not overwrite trunk. 
+2. When the workflow is complete, visit [https://wordpress.org/plugins/developers/releases/](https://wordpress.org/plugins/developers/releases/) and approve the release. It will take a few minutes for WordPress.org to build teh new version.
+3. Validate that the new release is available:
+   * https://plugins.svn.wordpress.org/woocommerce/tags/ should list the version number.
+   * The new version should also be available inside the Previous versions dropdown [inside the Advanced Options screen on the WordPress.org plugin page](https://wordpress.org/plugins/woocommerce/advanced/).
+4. If this is not an `RC` release, [Deploy the release to our staging environments](https://wp.me/PCYsg-18BQ) and monitor for any issues.
+5. Publish the previously created GitHub [draft release tag](https://github.com/woocommerce/woocommerce/releases).
+   * Check off Set as a pre-release if you’re releasing an RC. Otherwise, check off Set as the latest release.
+6. Ping Dev Advocacy in the #woo-core-releases channel in Slack notifying them of the release and asking them to update or publish the release post. Provide links to both the GitHub tag as well as the ZIP from WordPress.org as part of your message.
```

Comment:
I believe this is no longer necessary. Can you confirm @albarin?

Reply:
Should running the [`Release: Post-release published flows`](https://github.com/woocommerce/woocommerce/blame/trunk/.github/workflows/release-new-release-published.yml) replace this step?

Reply:
You mean step 6? I don't think it's that particular workflow but we do have some notifications going to Slack when a new release is marked as stable. I might be missing something, so I'll let @albarin confirm.

Reply:
We ask DevAdv to publish the release post with this https://github.com/woocommerce/woocommerce/blob/7fceed1befb9f456225cfed20298a1160075d666/.github/workflows/release-new-release-published.yml#L1-L22 and then we notify again when it's marked as stable in https://github.com/woocommerce/woocommerce/blob/7fceed1befb9f456225cfed20298a1160075d666/.github/workflows/release-update-stable-tag.yml#L235-L247

Reply:
Ok.  I removed the final two steps that mentioned sending slack notifications since they're covered.  I don't think it's a big deal, but the woo announcement slack ping may go out before the notice to DevAdv.  

One thing I think we're missing is updating the github release tag with a link to the release announcement.  But I think we should update that later when we go back through and clean things up.